### PR TITLE
[Merged by Bors] - feat(Prod.Basic/UniformSpace.Basic): Two lemmas about swap.

### PR DESCRIPTION
**Pull request by D-Thomine**
A special case  of lemma `map_comp_swap` in `Data.Prod.Basic` is restated in terms of `Function.Semiconj`.

~~Also, with little surprise, `swap` is uniformly continuous.~~

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by D-Thomine** [comment]
@j-loreaux : You have put the `awaiting-author` label. Is there anything I need to do on this PR?

**Comment by j-loreaux** [comment]
Oops, sorry, I had a pending comment.

File: Mathlib/Topology/UniformSpace/Basic.lean [hunk 1]
```diff
@@ -1441,6 +1441,9 @@ theorem toTopologicalSpace_prod {α} {β} [u : UniformSpace α] [v : UniformSpac
       @instTopologicalSpaceProd α β u.toTopologicalSpace v.toTopologicalSpace :=
   rfl
 
+theorem uniformContinuous_swap : UniformContinuous (Prod.swap : α × β → β × α) :=
+  uniformContinuous_snd.prod_mk uniformContinuous_fst
+
```
[the remark this answers is not in the document]
Comment (D-Thomine) [reply Mathlib/Topology/UniformSpace/Basic.lean]: Oh, I was no aware of that part. This is actually even better for what I want to do!

I've deleted `uniformContinuous_swap` ; all that remains is `Function.Semiconj.swap_map`.

**Comment by j-loreaux** [comment]
bors merge

**Pull request closed without merging**